### PR TITLE
Examples: Make position codes consistent in comments

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
**Description of proposed changes**

Currently, we have different versions for explaining the position / justification codes in the comments of the examples (Gallery, Tutorials, Intros). Let's take `TL` as example:
- top left
- Top Left
- TopLeft
- top-left

I feel we should decide on one version and use it consistently across all examples. I personally prefer TopLeft (or Top Left).
@GenericMappingTools/pygmt-maintainers what is your preferred version? After we decided for one version, I can update all examples regarding this. 

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.
- [ ] Use underscores (not hyphens) in names of Python files and directories.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash command is:

- `/format`: automatically format and lint the code

### CODE REVIEW

**Comment by weiji14** [comment]
> Top Left

Personally prefer "Top Left" (though not strongly tied to this). Maybe also good to include the abbreviation, so have sentences like `... in the Top Left (TL) corner ...`?

**Comment by seisman** [comment]
> Maybe also good to include the abbreviation, so have sentences like `... in the Top Left (TL) corner ...`?

i like this one.

**Comment by seisman** [comment]
Although middle-right can be specified as either `MR` or `RM`, I prefer Middle Right `MR`, i.e., Vertical code first and then Horziontal code.

File: examples/gallery/basemaps/double_y_axes.py [hunk 1]
```diff
@@ -63,7 +63,8 @@ class can control which axes should be plotted and optionally show annotations,
 # Plot points for y2 data
 fig.plot(x=x, y=y2, style="s0.28c", fill="red", label="y2")
 
-# Create a legend in the top-left corner of the plot
+# Add a legend in the Top Left (TL) corner of the plot with on
+# offset of 0.1 centimeters
```
Comment (yvonnefroehlich) [inline examples/gallery/basemaps/double_y_axes.py:67]:
```suggestion
# Create a legend in the Top Left (TL) corner of the plot with an
# offset of 0.1 centimeters
```

File: examples/gallery/embellishments/colorbar.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 - **j/J**: placed inside/outside the plot bounding box using any 2-character
   combination of vertical (**T**\ op, **M**\ iddle, **B**\ ottom) and
   horizontal (**L**\ eft, **C**\ enter, **R**\ ight) alignment codes, e.g.
-  ``position="jTR"`` for Top Right.
+  ``position="jTR"`` for TopRight.
```
Comment (yvonnefroehlich) [inline examples/gallery/embellishments/colorbar.py:17]:
```suggestion
  ``position="jTR"`` for Top Right.
```

File: examples/gallery/embellishments/colorbar.py [hunk 3]
```diff
@@ -34,7 +34,7 @@
 
 # ============
 # Create a colorbar designed for seismic tomography - roma
-# Colorbar is placed at bottom center (BC) by default if no position is given
+# Colorbar is placed at BottomCenter (BC) by default if no position is given
```
Comment (yvonnefroehlich) [inline examples/gallery/embellishments/colorbar.py:37]:
```suggestion
# Colorbar is placed at Bottom Center (BC) by default if no position is given
```

File: examples/gallery/embellishments/legend.py [hunk 4]
```diff
@@ -43,8 +43,8 @@
 fig.plot(x=x, y=y2, style="c0.07c", fill="dodgerblue", label="cos(x)+1.1")
 
 # Add a legend to the plot; place it within the plot bounding box with both
-# reference ("J") and anchor ("+j") points being TopRight and with an offset
-# of 0.2 centimeters in x and y directions; surround the legend with a box
+# reference ("J") and anchor ("+j") points being the Top Right corner and an
+# offset of 0.2 centimeters in x and y directions; surround the legend with a box
```
Comment (yvonnefroehlich) [inline examples/gallery/embellishments/legend.py:47]:
```suggestion
# reference ("J") and anchor ("+j") points being the Top Right (TR) corner and an
# offset of 0.2 centimeters in x and y directions; surround the legend with a box
```

File: examples/gallery/lines/wiggle.py [hunk 5]
```diff
@@ -35,8 +35,8 @@
     pen="1.0p",
     # Draw a blue track with a width of 0.5 points
     track="0.5p,blue",
-    # Plot a vertical scale bar at the right middle. The bar length is 100 in
-    # data (z) units. Set the z unit label to "nT".
-    position="jRM+w100+lnT",
+    # Plot a vertical scale bar at position Middle Right (MR). The bar length (+w)
```
Comment (yvonnefroehlich) [inline examples/gallery/lines/wiggle.py:38]:
```suggestion
    # Plot a vertical scale bar at position Middle Right (MR). The bar length (+w)
```

**Review by seisman — APPROVED** [review]

**Comment by yvonnefroehlich** [comment]
Currently building the documentation fails. The issue seem to be related to the [README file containing the projection table](https://github.com/GenericMappingTools/pygmt/tree/make-justify-codes-consistent/examples/projections/table).

**Pull request merged**